pkg/oc/admin/migrate/migrator.go

```diff
@@ -98,11 +98,10 @@ func (o *ResourceOptions) Bind(c *cobra.Command) {
 }
 
 func (o *ResourceOptions) Complete(f *clientcmd.Factory, c *cobra.Command) error {
-	// oc adm migrate authorization does not support printing and takes no parameters, so we ignore the error here
 	var flagErr error
```

Comment:
Remove the `flagErr` bits and just do `o.Output = kcmdutil.GetFlagString(c, "output")`.